src/pages/Chaehyun/SignUpBaeminSurvival.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import InputList from './InputList';
+import './SignUpBaeminSurvival.scss';
+
+const signUpData = [
```

Comment:
```suggestion
const SINGUP_DATA = [
```
상수데이터는 UPPER_CASE + snake_case가 정석입니다!

Reply:
상수데이터 사용하신것 너무 좋네요 👍 
다만, 변수명은 명확하게 의미가 드러나도록 지어주세요. 코드는 쓰여지는 것보다 읽히는 경우가 훨씬 더 많습니다. 이름만 보더라도 어떤 값을 담고 있는 변수인지 알 수 있게 지어주세요

Reply:
그리고 상수데이터가 먼저 나오기보다는 파일 최하단에 있는것이 좀 더 가독성에 좋을 것 같네요!